src/test/java/vehicle/PutVehicleTechnicalRecordsNeg.java

```diff
@@ -78,6 +78,7 @@ public void testErrorForMissingMandatoryAdrFields() {
         //TEST
         vehicleTechnicalRecordsSteps.postVehicleTechnicalRecordsWithAlterations(requestBodyHgv, alterations);
         vehicleTechnicalRecordsSteps.statusCodeShouldBe(201);
+        vehicleTechnicalRecordsSteps.waitForVehicleTechRecordsToBeUpdated(randomVin, 10);
```

Comment:
Shouldn't we put here 20 seconds like in the other tests where we use this step?